### ISSUE TITLE
Allow wildcard dependencies for compilation

### ISSUE THREAD

**Issue by beatgeb** [namics/generator-nitro#12]
If mixins or colors (in less, scss or similar) are splitted in multiple files it would be nice to address them in the config.json with the wildcard sign.
Example:
```
"app.css": [
			"+assets/css/variables.scss",
			"+assets/css/mixins.scss",
			"+assets/css/variables/*.scss",
			"+assets/css/mixins/*.scss",
			"assets/css/reset.css",
			"assets/css/basic.scss",
			"components/**/css/*.scss",
			"components/**/css/skins/*.scss"
		],
```
instead of 
```
"app.css": [
			"+assets/css/variables.scss",
			"+assets/css/mixins.scss",
			"+assets/css/variables/_baukasten.scss",
			"+assets/css/variables/_colors.scss",
			"+assets/css/variables/_constants.scss",
			"+assets/css/variables/_utils.scss",
			"+assets/css/mixins/_arrows.scss",
			"+assets/css/mixins/_border-dotted.scss",
			"assets/css/reset.css",
			"assets/css/basic.scss",
			"components/**/css/*.scss",
			"components/**/css/skins/*.scss"
		],
```

**Comment by deniaz** [comment]
@beatgeb can you verify that 4ff7b83 fixes this?

The issue's not completely solved for me, though. If you're already `watching` and add new dependencies to `config.json`, `gulp` will not import those.
I will look into this tomorrow and am hopeful to be able to provide a fix soon.

**Comment by deniaz** [comment]
@beatgeb config is now reloaded when changes.
Fixed by 035886e.
